crates/bevy_asset/src/processor/mod.rs

```diff
@@ -1,3 +1,25 @@
+//! Asset processing in Bevy is a framework by which artist-authored assets can be automatically prepared for use in game.
```

Comment:
```suggestion
//! Asset processing in Bevy is a framework for automatically crunching(?*) artist-authored assets into formats suitable for use in game.
```

*can't think of a good verb here

Reply:
```suggestion
//! Asset processing in Bevy is a framework for automatically transforming artist-authored assets into the format that best suits the needs of your particular game.
```

Reply:
Ehh still feel like we lost some intent here. I feel like it should mention "runtime" and maybe "optimize" somewhere.

Reply:
Perf-centric optimization definitely the most common application, but it's definitely not the only one, especially with a powerful and flexible system. I'd like to avoid tying the docs too tightly to that use case.